src/main/java/sa/assertj/maps/Contains.java

```diff
@@ -0,0 +1,41 @@
+package sa.assertj.maps;
+
+import sa.assertj.Experiment;
+import sa.assertj.Util;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Contains extends Experiment {
+
+    static Experiment.DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int s=0; s < numOfSamples; s++) {
+            Map<String, String> actual = new LinkedHashMap<>();
+            Map.Entry[] expected = new Map.Entry[size];
+            for (int i=0; i < size; i++) {
+                String key = Util.randomString(10);
+
+                if (actual.containsKey(key)) {
+                    i--;
+                    continue;
+                }
+
+                String value = Util.randomString(10);
+
+                actual.put(key, value);
+                expected[i] = Map.entry(key, value);
```

Comment:
I feel like we should randomly choose whether we add this in or not, to make sure we have some randomness in the amount being checked (cause this isn't `containsExactly` and thus doesn't need to contain all of them).

Reply:
Put it on 80% for now, we can change it with one variable